src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -22,35 +22,103 @@ CPU::CPU() {
     std::vector<std::vector<std::string>> system_info_table;
 
     _num_threads = parallel_get_max_threads();
-    auto GetCatchInfoLinux = [&]() {
-        _processors = sysconf(_SC_NPROCESSORS_ONLN);
-        system_info_table.resize(_processors, std::vector<std::string>(3));
+    auto GetCacheInfoLinux = [&]() {
```

Comment:
I think this name and "GetFreqInfoLinux" should be changed by under score case.

Reply:
updated